docker/images/alpine/dev/Dockerfile

```diff
@@ -15,19 +15,18 @@ RUN mkdir -p /data/{media,static}
 
 # Install curl
 RUN apk --no-cache add --update \
-        curl \
-        vim \
-        musl-dev  # required for Sphinx
+    curl \
+    vim \
+    musl-dev  # required for Sphinx
 
 # Install development dependencies
-RUN python -c "import configparser; c = configparser.ConfigParser(); c.read('src/backend/setup.cfg'); \
-               print(c['options.extras_require']['dev'])" | xargs pip install
+RUN pip install .[dev]
```

Comment:
goes beyond the commit message! Make it in a separate commit?

Reply:
True that!

Reply:
Done.